modules/get-started/pages/quick-start.adoc

```diff
@@ -36,111 +57,301 @@ docker compose version
+. Click *Create*.
+
+image::console:user.png[]
+
+You now have a new user called Sasha. But this user has no permissions yet.
+
+. In the *ACLs* tab, click *Create ACL*.
+. Enter "Sasha" in the user/principal field.
+. In the topic selector dropdown, select *Literal* and enter "transactions" in the input field.
+. Under *Operations*, click the *All* dropdown and select *Allow*.
+. Scroll down and click *OK*.
+
+Now Sasha has full access only to the topic called transactions.
+
+To test these permissions, you can use `rpk` to connect to Redpanda as the user Sasha.
+
+. Try to access the `logins` topic as Sasha:
 +
-.Reveal the YAML content
-[%collapsible]
-====
-.`docker-compose.yml`
-[,yaml,subs="attributes+"]
+[,bash]
 ----
-include::get-started:attachment$three-brokers/docker-compose.yml[]
+docker exec -it redpanda-0 rpk topic describe logins \
+  -X user=Sasha \
+  -X pass=sashapassword \
+  -X sasl.mechanism=SCRAM-SHA-256 \
+  -X brokers=redpanda-0:9092
 ----
-====
-
-. Run the following in the directory where you saved the `docker-compose.yml` file:
 +
-```bash
-docker compose up -d
-```
+You are not authorized to view the topic.
 +
 [.no-copy]
 ----
-[+] Running 8/8
-⠿ Network redpanda_network                  Created  0.0s
-⠿ Volume "redpanda-quickstart_redpanda-0"   Created  0.0s
-⠿ Volume "redpanda-quickstart_redpanda-1"   Created  0.0s
-⠿ Volume "redpanda-quickstart_redpanda-2"   Created  0.0s
-⠿ Container redpanda-0                      Started  0.3s
-⠿ Container redpanda-console                Started  0.6s
-⠿ Container redpanda-1                      Started  0.7s
-⠿ Container redpanda-2                      Started  0.7s
+SUMMARY
+=======
+NAME        logins
+PARTITIONS  0
+ERROR       TOPIC_AUTHORIZATION_FAILED: Not authorized to access topics: [Topic authorization failed.]
+
+CONFIGS
+=======
+config response contained error: TOPIC_AUTHORIZATION_FAILED: Not authorized to access topics: [Topic authorization failed.]
 ----
 
---
-======
+. Access the `transactions` topic.
++
+[,bash]
+----
+docker exec -it redpanda-0 rpk topic describe transactions \
+  -X user=Sasha \
+  -X pass=sashapassword \
+  -X sasl.mechanism=SCRAM-SHA-256 \
+  -X brokers=redpanda-0:9092
+----
++
+You have access to this topic.
+
+=== Data transforms
+
+Data transforms let you run common data streaming tasks, like filtering, scrubbing, and transcoding, on the Redpanda broker. For example, you may have consumers that require you to redact credit card numbers or convert JSON to Avro.
+
+This deployment comes with one transform function called `regex` running in your cluster. Its job is to find records in the `logins` topic that contain email addresses with the `.edu` domain and add those to a new topic called `edu-filtered-domains`.
+
+In the left menu, click *Transforms*.
+// (step {"action":"find", "selector": "[data-testid='nav-link-Transforms']", "matchText": "Transforms", "click": true})
+
```

Comment:
@JakeSCahill I don't really understand what's happening here. I was going to edit to, "On the **Transforms** page, you see your transform." But is this config required for doc detective?

Reply:
Only lines that start with comments like `//` are for Doc Detective.